### ISSUE TITLE
Table of contents and title page not rendered correctly

### ISSUE THREAD

**Issue by Azer0s** [asciidoctor/asciidoctor-pdf#1012]
For this input:

```
= Microservice Architecture and Distributed Systems
Ariel Simulevski <ariel@simulevski.at>
v0.1, 2019-03-24
:toc: macro
:math:
:media: prepress
:icons: font
:source-highlighter: rouge
:stem: latexmath

toc::[]

<<<
```

I am getting this result: 
![Capture](https://user-images.githubusercontent.com/16290284/54884439-64beaf00-4e71-11e9-8864-ac78fd14bd38.PNG)

Instead of a title page and then a toc.
I am using WSL and using `Asciidoctor PDF 1.5.0.alpha.17.dev using Asciidoctor 1.5.8`.

This is the command I use to build the PDF: 
`asciidoctor-pdf -r asciidoctor-mathematical -r asciidoctor-diagram src/main/asciidoc/diplomathesis.adoc`

**Comment by mojavelinux** [comment]
If you're using the toc with the article doctype, you need to set the `title-page` attribute to restore the previous behavior. (We're still working a better default for this, see #233).

```
= Microservice Architecture and Distributed Systems
Ariel Simulevski <ariel@simulevski.at>
v0.1, 2019-03-24
:title-page:
...
```

Keep in mind that Asciidoctor PDF does not currently support arbitrary placement of the toc.

**Comment by mojavelinux** [comment]
This has been resolved by the fix for #233. As such, I'll mark this as a duplicate.

**Comment by mojavelinux** [comment]
FYI, prepress is only relevant if the doctype is book. You also should avoid using manual page breaks (unless you're trying to get behavior not supported by doctype=book media=prepress).
